docs/application/native/guides/notification/notifications.md

```diff
@@ -99,11 +99,25 @@ A status message type notification can be used like a pop-up. It is displayed fo
 
 The following notification layouts are provided:
 
-- `NOTIFICATION_LY_NOTI_EVENT_SINGLE`Layout for a single event notification.
-- `NOTIFICATION_LY_NOTI_EVENT_MULTIPLE`Layout for a multiple event notification.
-- `NOTIFICATION_LY_NOTI_THUMBNAIL`Layout for a notification displaying images.
-- `NOTIFICATION_LY_ONGOING_PROGRESS`Layout for an ongoing notification displaying progress.
-- `NOTIFICATION_LY_EXTENSION`Layout for an [extended notification](#extended_noti).
+- `NOTIFICATION_LY_NOTI_EVENT_SINGLE`
+
+   Layout for a single event notification.
+
+- `NOTIFICATION_LY_NOTI_EVENT_MULTIPLE`
+
+   Layout for a multiple event notification.
+
+- `NOTIFICATION_LY_NOTI_THUMBNAIL`
+
+   Layout for a notification displaying images.
+
+- `NOTIFICATION_LY_ONGOING_PROGRESS`
+
+   Layout for an ongoing notification displaying progress.
+
+- `NOTIFICATION_LY_EXTENSION`
+
+   Layout for an [extended notification](#extended_noti).
 
 **Figure: Notification layouts**
 
```

Comment:
line 263: Add a space. (in the _notification_event_typeenumeration)

Reply:
done